pkg/network/http/model_windows.go

```diff
@@ -0,0 +1,119 @@
+// +build windows,npm
+
+package http
+
+import (
+	"encoding/binary"
+	"errors"
+
+	"github.com/DataDog/datadog-agent/pkg/network/driver"
+)
+
+const HTTPBufferSize = driver.HttpBufferSize
+const HTTPBatchSize = driver.HttpBatchSize
+
+type httpTX driver.HttpTransactionType
+
+// errLostBatch isn't a valid error in windows
+var errLostBatch = errors.New("invalid error")
+
+// Path returns the URL from the request fragment captured in the driver with GET variables excluded.
+// Example:
+// For a request fragment "GET /foo?var=bar HTTP/1.1", this method will return "/foo"
+func (tx httpTX) Path(buffer []byte) []byte {
+	b := tx.RequestFragment
+
+	var start, end int
+	for start = 0; start < len(b) && b[start] != ' '; start++ {
```

Comment:
See the gnuTLS PR for an example of how to solve this, but we need to handle a null byte before the end of the buffer. 

Also, this logic looks very similar to the Linux side. Is there a difference, or could we combine them?

Reply:
The logic is basically identical to the Linux side, but the types are different (`[25]int8` on windows vs `[25]byte`), so on Linux we can use `copy` and on windows we have to copy & cast each byte ourselves.

Reply:
I realize this would require a driver change, but could we use an unsigned char type in the driver so both Linux and Windows could use `byte`? I think the `BYTE` type is available: https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types